tests/misalignedTests.js

```diff
@@ -0,0 +1,14 @@
+const {expect} = require('chai');
+const {colorMap}=require('../misaligned');
+
+expect(colorMap.length).equals(25);
+
+colorMap.forEach((obj)=>{
```

Comment:
Just calling out the manual vs automated checks here:
- The code below guarantees that the colors and the numbers will be in range
- Manual inspection is needed to determine that duplicate+missing rows are not there
- Manual inspection is also needed to guarantee the alignment, presence of header, etc.
Typically, we take a call if this is ok, by judging how frequently these things can change.